web/packages/core/src/ruffle-player.ts

```diff
@@ -772,10 +780,18 @@ export class RufflePlayer extends HTMLElement {
     }
 
     private pointerDown(event: PointerEvent): void {
+        if (event.pointerType === "mouse" && event.button === 2) {
+            this.showContextMenu.bind(this)(event);
```

Comment:
The `.bind(this)` is totally redundant (here and in other places).

Reply:
Ah right, here's some more info about that: https://javascript.info/bind
Basically it's only necessary when the function is being passed to something outside the object, like the setTimeout function.

Reply:
Wasn't sure about this. I'll remove these instances.